projects/ngneat/transloco-locale/src/lib/tests/pipes/transloco-date.pipe.spec.ts

```diff
@@ -80,6 +81,9 @@ describe('TranslocoDatePipe', () => {
 
   it('should transform an ISO 8601 string to date', () => {
     const pipe = new TranslocoDatePipe(service, cdr, LOCALE_CONFIG_MOCK);
-    expect(pipe.transform('2019-09-12T19:51:33Z')).toEqual('Sep 12, 2019, 10:51:33 PM');
```

Comment:
@chrsi Can you please add the timezone property to the `LOCALE_CONFIG_MOCK` instead of adding this code?

Reply:
Do you mean extracting line 85-86 to a function in the mocks.ts file?

Reply:
There isn't really a timezone property in place, because the expected and tested value are both based on UTC.

Reply:
maybe I'm missing something, what was the output before your change? and what was the cause?

Reply:
So '2019-09-12T19:51:33Z' is a date formatted in ISO 8601 representing 'Sep 12, 2019, 7:51:33 PM' (because it's represented as UTC due to the trailing Z). Once I try to transform/evaluate that datetime my machine will convert it to my current timezone (which is +2 hours) making it 'Sep 12, 2019, 9:51:33 PM'. Since you're probably running the tests in a timezone one ours ahead of mine (+3 hours) the test works fine, because the date will be transformed to 'Sep 12, 2019, 10:51:33 PM'. But for everyone that isn't in your timezone, the tests will fail.

What my code does is keeping the transformed date as is (because I'm not sure whether transloco is currently capable of setting the timezone used for interpreting the datetime) but changing the expected value based on the time zone. With my code the expected value isn't fixed to '10:51:33 PM' anymore but rather calculated from the UTC datetime under test (see `Date.UTC(2019, 8, 12, 19, 51, 33)`) on the machine that is running the tests (making it timezone aware).

I hope I could explain it clear enough :)
An alternative solution would be to make the value under test timezone aware. But I'm not sure how (or if it's possible) to set the timezone information in transloco.

Reply:
Oh ... as a matter of fact, I just found out that transloco is already capable of that. The value under test is now interpreted as a UTC datetime (because the provided ISO 8601 value uses UTC). This should make the test pass in all timezones.
Thanks for hinting me to the solution. 👍